### DISCUSSION TITLE
Custom GraphQL queries - how do I reference the DateTime type?

### DISCUSSION THREAD

**Discussion by tyteen4a03** [payloadcms/payload#8882] [Q&A]
What is the correct type for the `DateTime` type in the schema? I want to refer to it in the custom schema.

Trying this leads to `Error: Schema must contain uniquely named types but contains multiple types named "DateTime".`

```
    args: {
        departureTime: {
            type: GraphQLDateTime, // comes from import { GraphQLDateTime } from "graphql-scalars";
        },
    }
```

**Comment by fede-s** [comment]
I have this same problem but with GraphQLJSON, it was ok on payloadcms v2, it became an issue after trying to migrate to v3. @tyteen4a03 Were you able to fix this?

```
import { GraphQLObjectType, GraphQLString } from 'graphql/type';
import { GraphQLJSON } from 'graphql-type-json';

export const TripHost = new GraphQLObjectType({
  name: 'TripHost',
  fields: {
    name: {
      type: GraphQLString,
    },
    slogan: {
      type: GraphQLString,
    },
    description: {
      type: GraphQLJSON,
    },
  },
});
```

**Reply by tyteen4a03** [reply]
No, was hoping @DanRibbens would have a better solution!

I gave up and used Strings instead.

**Reply by fede-s** [reply]
They have created their own scalar for `JSON` and named it the same as the one on `graphql-type-json`. So when importing `GraphQLJSON` from `graphql-type-json` I end up having 2 scalars named `JSON`.
So the fix is using their type:

```
import { GraphQLJSON } from '@payloadcms/graphql/types';
```
